lib/odoobaselib/__init__.py

```diff
@@ -57,6 +58,15 @@ def addons_config():
                     for glob in addons:
                         logging.debug("Expanding glob %s", glob)
                         for addon in iglob(os.path.join(SRC_DIR, repo, glob)):
+                            if not os.path.isdir(addon):
+                                continue
+                            addon_files = set(os.listdir(addon))
```

Comment:
I feel that `if not (os.path.isfile(os.path.join(addon, "__manifest__.py")) or os.path.isfile(os.path.join(addon, "__openerp__.py"))):` would be faster, don't you think? 🤔

Reply:
Indeed, right you are.